gemset.nix

```diff
@@ -0,0 +1,253 @@
+{
```

Comment:
Was checking in this file on purpose?

Reply:
Absolutely. I have another branch for it which I was wondering if I should make into a different PR. Not having it means I can't compile the site, which was really very frustrating.